sos/report/plugins/networking.py

```diff
@@ -183,14 +183,23 @@ def setup(self):
                 "ethtool -a " + eth,
                 "ethtool -c " + eth,
                 "ethtool -g " + eth,
-                "ethtool -e " + eth,
                 "ethtool -P " + eth,
                 "ethtool -l " + eth,
                 "ethtool --phy-statistics " + eth,
                 "ethtool --show-priv-flags " + eth,
                 "ethtool --show-eee " + eth
             ])
 
+            # skip EEPROM collection for 'bnx2x' driver NICs as this command 
+            # can pause the NIC and is not production safe.
+            cmd="ethtool -i {}".format(eth)
+            driver = self.get_command_output(cmd)
```

Comment:
`get_command_output()` no longer exists in the sos API. You need to use `exec_cmd()` if you need to run a command and get its output during `setup()`.